src/extension/packagerStatusIndicator.ts

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import {window, Disposable, StatusBarItem, StatusBarAlignment} from "vscode";
+
+/**
+ * Updates the Status bar with the status of React Native Packager.
+ */
+
+export enum PackagerStatus {
+    PACKAGER_STARTED = 0,
+    PACKAGER_STOPPED
+}
+
+export class PackagerStatusIndicator implements Disposable {
+    private packagerStatusItem: StatusBarItem;
+    private static PACKAGER_STARTED_STATUS_STR: string = "React Native Packager: Started";
+    private static PACKAGER_STOPPED_STATUS_STR: string = "React Native Packager: Stopped";
+
+    public constructor() {
+        this.packagerStatusItem = window.createStatusBarItem(StatusBarAlignment.Left);
+    }
+
+    public dispose(): void {
+        this.packagerStatusItem.dispose();
+    }
+
+    public updatePackagerStatus(status: PackagerStatus): void {
```

Comment:
At the moment the parameter is always hard-coded on the call site. I think this would be simpler if we just used two methods: packagerWasStarted() packagerWasStopped() rather than this method with a parameter/switch.